rollouts/docs/running-locally.md

```diff
@@ -0,0 +1,176 @@
+# Running Rollouts Locally
+
+## Prerequisites
+
+To run Rollouts locally, you will need:
+
+* Linux machine (technically it is possible to run Rollouts locally on a Mac but
+  due to differences in Docker between Linux and Mac, the Rollouts scripts are
+  confirmed to work on Linux)
+* [go 1.19](https://go.dev/dl/) or newer
+* `make`
+* Access to GKE clusters.
+
+## Cluster Setup
+
+### Creating a management cluster with KCC running
+First, you must create a config-controller cluster. You can follow the instructions on the 
+[config-controller-setup guide](https://cloud.google.com/anthos-config-management/docs/how-to/config-controller-setup) to create a config-controller cluster. Make sure your kubeconfig is 
+connected to this cluster.
+
+### Provisioning child clusters
+The next step will be to provision new child clusters. This example names the child cluster `gke-n`;
+you can replace `gke-n` with whatever name you want for your child cluster. 
+
+```sh
+# first step is to create a deployable instance of the package
+kpt pkg get git@github.com:droot/kpt-packages.git/cluster@main gke-n --for-deployment
+
+# now render the package
+kpt fn render
+
+# assuming your kubectl is configured to talk to a management cluster with KCC running (the previous step).
+# provision the cluster using following commands
+kpt live init gke-n
+kpt live apply gke-n
+```
+
+You can repeat the above steps to create as many child clusters as you want.
+
+### Setting up kubeconfig
+Next, we will configure kubeconfig to be able to talk to each cluster individually.
+
+```sh
+# once kpt live status is showing current for a cluster package
+# run the following
+KUBECONFIG=~/.kube/gke-n gcloud container clusters get-credentials gke-n --region us-west1
+
+# now ~/.kube/gke-n file has been updated with the credentials for gke-n cluster
+# verify if this is working
+
+KUBECONFIG=~/.kube/gke-n kubectl get pods -n kube-system
+```
+
+### Set up Config Sync
+Rollouts requires a git syncer installed on child clusters. Currently, only Config Sync is supported. 
+To install Config Sync on your child clusters:
+
+```sh
+kpt pkg get git@github.com:droot/kpt-packages.git/config-sync@main config-sync
```

Comment:
We can probably use the manifests from the `kpt-config-sync` project directly.

Reply:
SG. I updated this to instead follow the instructions on https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/docs/installation.md